debian/changelog

```diff
@@ -1,8 +1,278 @@
-mutter (3.32.1-1endless0) eos; urgency=medium
```

Comment:
On gsd rebase I've kept the old endless entries for reference, not sure if worth doing the same here.

Reply:
We don't usually leave the previous changelog entries when rebases.

Reply:
Ack, I've updated https://github.com/endlessm/gnome-settings-daemon/pull/59 to remove the old entries as well.